### PR TITLE
Android porting

### PR DESCRIPTION
**Pull request by menaganti**
Added native build
Successful compilation and build for android.
Note: kinit stalls, need to debug.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -59,6 +59,8 @@ asn1_*.[cx]
 /tags
 /test-driver
 /ylwrap
+/build
```
Comment (nicowilliams) [inline .gitignore]: I'd rather not ignore this.  I like being able to do `git clean -fdx` to get a clean workspace.

File: appl/test/http_client.c [hunk 2]
```diff
@@ -78,7 +78,7 @@ do_connect (const char *hostname, const char *port)
 
     return s;
 }
-
+/*
```
Comment (nicowilliams) [inline appl/test/http_client.c]: What's this about?

File: lib/com_err/compile_et.c [hunk 3]
```diff
@@ -109,16 +109,16 @@ generate_c(void)
 	    "void initialize_%s_error_table_r(struct et_list **list)\n",
 	    name);
     fprintf(c_file, "{\n");
-    fprintf(c_file,
+    /*fprintf(c_file,
```
Comment (nicowilliams) [inline lib/com_err/compile_et.c]: What's this about?

**Comment by nicowilliams** [comment]
@menaganti Why close it?  It's a nice contribution (though, naturally, code review comments need addressing).

**Comment by menaganti** [comment]
It's incomplete. I have addressed some compilation issues and made the
kinit work against android bionic for arm . However, I feel that it is not
enough though. I want to clean up and address some issues inherent.

~Manoj

**Pull request closed without merging**